RecoEgamma/PhotonIdentification/plugins/PhotonIDValueMapProducer.cc

```diff
@@ -378,10 +418,17 @@ void PhotonIDValueMapProducer::produce(edm::Event& iEvent, const edm::EventSetup
       if( thisCandidateType == reco::PFCandidate::h ){
 	// for charged hadrons, additionally check consistency
 	// with the PV
-
-        auto dxydz = getTrackDxyDz(iCand, pv.position(), isAOD);
-        if ( fabs(dxydz.first) > dxyMax) continue;
-        if ( fabs(dxydz.second) > dzMax) continue;
+	float dxy = -999, dz=-999;
+        getImpactParameters(iCand, isAOD, pv, dxy, dz);
+
+	//
+	//const reco::Track *theTrack = getTrackPointer( iCand, isAOD );
+	//
+	//float dxy = theTrack->dxy(pv.position());
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

Reply:
Changes implemented here:
https://github.com/lsoffi/cmssw/blob/CMSSW_9_4_0_pre3_TnP/RecoEgamma/PhotonIdentification/plugins/PhotonIDValueMapProducer.cc